testnet/Dockerfile.debug

```diff
@@ -0,0 +1,46 @@
+# BUILD IMAGE ---------------------------------------------------------
+
+FROM rust:1.85.1-slim-bookworm AS builder
+
+# Install dependencies needed for building RocksDB and etcd.
+RUN apt-get update && apt-get install -yq \
+    git gcc g++ clang libssl-dev pkg-config ca-certificates curl cmake ninja-build
+
+COPY . /nomos
+
+WORKDIR /opt
+
+RUN git clone https://github.com/risc0/risc0.git && \
+    cd risc0 && \
+    cargo install --path rzup && \
+    rzup build rust --tag-or-commit 4d91de4e48198da2e33413efdcd9cd2cc0c46688 && \
+    git checkout origin/release-2.0 && \
+    cargo install --path risc0/cargo-risczero && \
+    rm -rf /usr/local/cargo/registry /usr/local/cargo/git && \
+    rm -rf /root/.risc0/tmp && \
+    rm -rf * && \
+    cd /nomos && \
+    cargo build && \
+    find . -type f ! -path "./target/debug/nomos-node" ! -path "./target/debug/nomos-executor" \
+    ! -path "./target/debug/nomos-cli" ! -path "./target/debug/cfgsync-server" ! -path "./target/debug/cfgsync-client" -delete
+
+
+FROM rust:1.85.1-slim-bookworm AS final
+
+LABEL maintainer="augustinas@status.im" \
```

Comment:
Is this true? 🙈

Reply:
Feel free to propose different maintainer.

Reply:
I guess this should either be you or be completely removed. What do you think? Unless this was taken and adapted with minor modifications from another `Dockerfile`?

Reply:
I've set myself as maintainer for the time being.
https://github.com/logos-co/nomos/pull/1212/commits/39d29ab7b96b0b3f58fe1a375e5145d45920b4cf